rna/rna-collect-multiple-metrics/README.md

```diff
@@ -0,0 +1,28 @@
+# RNACollectMultipleMetrics with Picard
+
+This workflow runs `picard CollectInsertSizeMetrics`, `picard CollectAlignmentSummaryMetrics`, `picard QualityScoreDistribution`, and `picard MeanQualityByCycle` on a BAM file
+(https://software.broadinstitute.org/gatk/documentation/tooldocs/4.0.1.2/picard_analysis_CollectInsertSizeMetrics.php).
+
+Inputs:
+- Per-sample:
+  - Sample name - Name of the sample associated with the BAM - used in names of outputs.
+  - BAM file - Path to a BAM file.
+  - Ref FASTA file - Reference Sequence FASTA file.
+- VM configuration
+  - Docker image url
+  - VM disk size
+  - Num CPU cores
+  - VM memory
+  - Runtime zones, ex: "us-central1-a us-central1-b"
+  - Number of times to try the workflow with a preemptible VM before
+    falling back to a full-price VM.
+
+Outputs:
+- Per-sample:
+  - File outputFiles - glob(*)
```

Comment:
Seems like the asterisk here is making github display the rest of the file in italics. I think you can safely ignore the fact that its displayed here in italics.

Reply:
Let's list all of the output files:

- &lt;sample-id&gt;.Multiple_Metrics.alignment_summary_metrics
- &lt;sample-id&gt;.Multiple_Metrics.base_distribution_by_cycle.pdf
- &lt;sample-id&gt;.Multiple_Metrics.base_distribution_by_cycle_metrics
- &lt;sample-id&gt;.Multiple_Metrics.insert_size_histogram.pdf
- &lt;sample-id&gt;.Multiple_Metrics.insert_size_metrics
- &lt;sample-id&gt;.Multiple_Metrics.quality_by_cycle.pdf
- &lt;sample-id&gt;.Multiple_Metrics.quality_by_cycle_metrics
- &lt;sample-id&gt;.Multiple_Metrics.quality_distribution.pdf
- &lt;sample-id&gt;.Multiple_Metrics.quality_distribution_metrics